crates/bevy_window/src/window.rs

```diff
@@ -283,6 +292,24 @@ impl Window {
     pub fn set_physical_cursor_position(&mut self, position: Option<DVec2>) {
         self.internal.physical_cursor_position = position;
     }
+
+    /// The window's current theme
+    ///
+    /// ## Platform-specific
+    ///
+    /// - iOS / Android / Web / x11: Unsupported.
```

Comment:
There is a bit of an issue here.

On unsupported platforms, winit always returns `None` when using `window.theme()`. However, in this bevy implementation, it's possible to set `self.internal.window_theme` to `Some(Theme)` in `set_window_theme`. Now this will return a misleading value.

I suggest conditionally returning `None` here, something like

```rust
if cfg!(any(target_os = "ios", target_os = "android", target_family = "wasm", feature = "x11")) {
  None
} else {
  self.internal.window_theme
}
```